logfields/common.go

```diff
@@ -0,0 +1,25 @@
+package logfields
```

Comment:
> Also open to having this in a new library but I feel we have shared everywhere anyways so just adding another light package to it feels fine.

How do you feel about refactoring shared a bit so that everything is in its own package under pkg folder.
My biggest quarrel with shared right now is that if I just want settings I have to also pull in a bunch of other dependencies I am not using.